Source/Src_nd/compute_dt.F90

```diff
@@ -58,39 +61,43 @@ subroutine estdt(lev, dt, umax, lo, hi, dx, &
     spdz    = 0.d0
     spdr    = 0.d0
     umax    = 0.d0
+
     !
     ! Limit dt based on velocity terms.
     !
     do k = lo(3), hi(3)
        do j = lo(2), hi(2)
           do i = lo(1), hi(1)
-             spdx = max(spdx ,abs(u(i,j,k,1)))
+             call amrex_max(spdx, abs(u(i,j,k,1)))
```

Comment:
This approach will, I think, not work on the GPU -- we'll need to rethink this. amrex_max is intended to be used for atomic updates to data in global memory, not local variables like spdx, spdy, spdz.